css/components/creativity.css

```diff
@@ -48,10 +46,22 @@ section.creativity > .row > .col-xs-12 {
     margin-top: 15px;
     text-align: center;
 }
-#creat-icons {
-  background: url(/images/hero/bg.jpg);
+#cIcons {
```

Comment:
CSS **labai** nerekomenduoju naudoti ID, nes po to bus **labai** sunki tokius selektorius permušti -> keičiam į klasę 👍